app/code/AKisilenko/ModuleLesson6/Model/AskQuestion.php

```diff
@@ -10,33 +10,134 @@
 use Magento\Framework\Model\ResourceModel\AbstractResource;
 use Magento\Framework\Registry;
 use Magento\Store\Model\StoreManagerInterface;
+use AKisilenko\ModuleLesson6\Api\Data\AskQuestionInterface;
 
 /**
  * Class AskQuestion
  * @package AKisilenko\ModuleLesson6\Model
- *
- * @method int|string getAskId()
- * @method int|string getName()
- * @method AskQuestion setName(string $name)
- * @method string getEmail()
- * @method AskQuestion setEmail(string $email)
- * @method string getTelephone()
- * @method AskQuestion setTelephone(string $phone)w
- * @method string getComment()
- * @method AskQuestion setComment(string $question)
- * @method string getRequest()
- * @method AskQuestion setRequest(string $request)
- * @method string getCreatedAt()
- * @method string getStatus()
- * @method AskQuestion setStatus(string $status)
- * @method int|string getStoreId()
- * @method AskQuestion setStoreId(int $storeId)
  */
-class AskQuestion extends AbstractModel
+class AskQuestion extends AbstractModel implements AskQuestionInterface
 {
     const STATUS_PENDING = 'pending';
     const STATUS_PROCESSED = 'processed';
 
+    public function getId()
+    {
+        return $this->getData('request_id');
```

Comment:
for some reason in your database the field name was ask_id, but not request_id:
return $this->getData('ask_id');